cmd/describe/cluster/cmd.go

```diff
@@ -70,6 +70,15 @@ func run(cmd *cobra.Command, argv []string) {
 	cluster := r.FetchCluster()
 	isHypershift := cluster.Hypershift().Enabled()
 
+	displayName := ""
+	subscription, subscriptionExists, err := r.OCMClient.GetSubscriptionBySubscriptionID(cluster.Subscription().ID())
+	if err != nil {
+		r.Reporter.Debugf("Failed to get subscription by ID: %s", err)
+	}
+	if subscriptionExists {
+		displayName = subscription.DisplayName()
```

Comment:
I don't mind doing this in the CLIs/UI. But if all clients will eventually be expected to do so, isn't it easier to retrieve as part of the ocm cluster spec in the response?

Reply:
- Would that require an extra CS->AMS call overhead?
  Especially when you **list** clusters in CS, does it already ask AMS for all matching subscriptions?

  fwiw, UI talks to both services anyway, for many fields.  (It can only talk to AMS for Deprovisioned clusters.). So it'd only help the CLI, UI wouldn't gain, and if it costs CS->AMS call(s), the system could become slower.

- If you add a field to CS api, ideally it should support `search` by it (very hard if source of truth remains AMS),
- and to patch it (doable).

These are not mandatory of course, but if you don't then was it worth it?  It'd remain simpler to use the more complete AMS API for all needs.

----

I think many of these assumptions are gonna be re-evaluated for XCM effort; until then this PR is good step imho.

Reply:
You mean have CS retrieve from AMS? I think @cben has context about the history of this idea.

Reply:
Going to resolve this for now, to merge this in. I would suggest having this conversation on a DDR to capture how to handle this for all clients. As this has and will be an on going topic of conversation. Would be good to have it documents and agreed too :crossed_fingers: